element/test/index.js

```diff
@@ -1,86 +1,81 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
-
-/**
- * Internal dependencies
- */
 import { createElement, renderToString, concatChildren, switchChildrenNodeName } from '../';
```

Comment:
This section should now be titled `Internal dependencies`, not `External dependencies`.